### PR TITLE
Add WOSPM checker project

### PR DESCRIPTION
**Pull request by umutphp**
The repo is under [WOSPM](https://github.com/WOSPM) organization and It is for auditing open source projects in terms of contribution friendly.

### CODE REVIEW

File: projects/WOSPM/checker.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: WOSPM Checker
+repoUrl: https://github.com/WOSPM/checker
+description: A checker for project owners to measure their open source project if it is a welcoming project for contributors or not.
+languages:
+  - PHP
+tags:
+  - First Timer Friendly
+  - Tools
+avatar: wospm-checker-avatar.png
```
Comment (jleven) [inline projects/WOSPM/checker.mdx:10]: Hi @umutphp, thanks so much for adding WOSPM checker -- it looks like a really cool project!

One thing, I'm not seeing this wospm-checker-avatar.png file. You can add it right in the /projects/WOSPM folder when you have a chance.

Comment (umutphp) [reply projects/WOSPM/checker.mdx]: @jleven Many tahns for the feedback :).

I have removed the avatar field from the mdx file.

**Review by deammer — APPROVED** [review]
Thank you @umutphp! This will be the first PHP project on OSS Port 😎

**Comment by umutphp** [comment]
@deammer 
Great :).

Many thanks for the merge and the great efforts to create the OSS Port.

**Pull request merged**